src/arch/xtensa/drivers/timer.c

```diff
@@ -16,32 +16,19 @@ struct timer_data {
 	void *arg2;
 };
```

Comment:
@tlauda please make this change also for imx: src/platform/imx8/include/platform/drivers/timer.h

Working on multiple platforms comes with the responsability of keeping the code consistent and a change should cover all existing platforms.

Reply:
@dbaluta It's done for IMX. I'm always covering all of the platforms.

Reply:
@tlauda Sorry, didn't notice that :)